docs/how to/use-transferbench.rst

```diff
@@ -1,106 +1,130 @@
-                  Character indicating memory type followed by device index (0-indexed)
-                  Supported memory locations are:
-                  - C:    Pinned host memory       (on NUMA node, indexed from 0 to [NUMA nodes-1])
-                  - U:    Unpinned host memory     (on NUMA node, indexed from 0 to [NUMA nodes-1])
-                  - B:    Fine-grain host memory   (on NUMA node, indexed from 0 to [NUMA nodes-1])
-                  - G:    Global device memory     (on GPU device indexed from 0 to [GPUs - 1])
-                  - F:    Fine-grain device memory (on GPU device indexed from 0 to [GPUs - 1])
-                  - N:    Null memory              (index ignored)
+   * - Transfers
+     - Number of transfers to be run in parallel
 
-Examples:::
+   * - SE
+     - Number of SEs to use (CPU threads or GPU threadblocks)
 
-   1 4 (G0->G0->G1)                   Uses 4 CUs on GPU0 to copy from GPU0 to GPU1
-   1 4 (C1->G2->G0)                   Uses 4 CUs on GPU2 to copy from CPU1 to GPU0
-   2 4 G0->G0->G1 G1->G1->G0          Copes from GPU0 to GPU1, and GPU1 to GPU0, each with 4 SEs
-   -2 (G0 G0 G1 4 1M) (G1 G1 G0 2 2M) Copies 1Mb from GPU0 to GPU1 with 4 SEs, and 2Mb from GPU1 to GPU0 with 2 SEs
+   * - srcMemL
+     - Source memory locations (Where the data is read)
 
-Round brackets and arrows' ->' may be included for human clarity, but will be ignored and are unnecessary
+   * - Executor
+     - | Executor is specified by a character indicating type, followed by the device index (0-indexed)
+       | - C: CPU-executed  (indexed from 0 to NUMA nodes - 1)
+       | - G: GPU-executed  (indexed from 0 to GPUs - 1)
+       | - D: DMA-executor  (indexed from 0 to GPUs - 1)
+
+   * - dstMemL
+     - Destination memory locations (where the data is written)
+
+   * - bytesL
+     - | Number of bytes to copy (use command-line specified size when 0)
+       | Must be a multiple of 4 and may be suffixed with ('K','M', or 'G')
+       | Memory locations are specified by one or more device characters or device index pairs.
+       | Characters indicate memory type and are followed by device index (0-indexed)
+       | Here are the characters and their respective memory locations:
+       | - C:    Pinned host memory       (on NUMA node, indexed from 0 to [NUMA nodes-1])
+       | - U:    Unpinned host memory     (on NUMA node, indexed from 0 to [NUMA nodes-1])
+       | - B:    Fine-grain host memory   (on NUMA node, indexed from 0 to [NUMA nodes-1])
+       | - G:    Global device memory     (on GPU device, indexed from 0 to [GPUs - 1])
+       | - F:    Fine-grain device memory (on GPU device, indexed from 0 to [GPUs - 1])
+       | - N:    Null memory              (index ignored)
+
+Round brackets and arrows "->" can be included for human clarity, but will be ignored.
 Lines starting with will be ignored. Lines starting with will be echoed to output
```

Comment:
@gilbertlee-amd Lines starting with what?

Reply:
Should be '#' for ignored lines
and '##' for echoed lines

Reply:
This is word-for-word with here:
https://github.com/ROCm/TransferBench/blob/a7315688c50d84af9cf1f9f100bc0b12d51f0b4c/examples/example.cfg#L61

Reply:
Done.